ibis/backends/bigquery/compiler.py

```diff
@@ -690,3 +690,44 @@ def visit_DropColumns(self, op, *, parent, columns_to_drop):
         table = sg.to_identifier(parent.alias_or_name, quoted=quoted)
         column = sge.Column(this=star, table=table)
         return sg.select(column).from_(parent)
+
+    def visit_TableUnnest(
+        self, op, *, parent, column, offset: str | None, keep_empty: bool
+    ):
+        quoted = self.quoted
```

Comment:
For my own education, what does `quoted` mean in this context?

Reply:
It's a flag to indicate whether to quote everything.

It's a bit weird, because it's currently set to `True` for every backend, but that was from a time when I thought we might be able to not quote everything.

It turned out to be much easier to just quote everything all the time.